api/src/repositories/messageboard/messageboard.repository.ts

```diff
@@ -36,11 +36,20 @@ export class MessageboardRepository {
       )
       .from<Member, Member[]>('member')
       .where('id', memberId);
+    const securityinfo = await knex
+      .select(
+        'member_id',
+      )
+      .from<RoleAssignment, RoleAssignment[]>('role_assignment')
+      .where('role_id', '62')
```

Comment:
We cannot put an id number in directly as your id for security chief maybe different than mine or ctr's.  We fix this by using a rolemap.  If you look at colony services, you will see an example.

Reply:
Changed it to where it is using rolemap as requested.